Type error in create-map/index.d.ts file

The `create-map/index.d.ts` causes the following error in my repo:
```
node_modules/nanostores/create-map/index.d.ts:2:38 - error TS2322: Type 'K' is not assignable to type 'string | number | symbol'.
  Type 'K' is not assignable to type 'symbol'.

2 type Get<T, K> = Extract<T, { [K1 in K]: any }>[K];
```

The same error I see in TypeScript playground ([here](https://www.typescriptlang.org/play?#code/C4TwDgpgBA4hwB4AqAaKBpAfFAvFAogB7ABOAhgMaKpQDeUA2ugIxQCWAdhgLoBcUZDiCgBfTE24BuAFBA)).

--- a/create-map/index.d.ts
+++ b/create-map/index.d.ts
@@ -1,5 +1,5 @@
 type AllKeys<T> = T extends any ? keyof T : never
-type Get<T, K> = Extract<T, { [K1 in K]: any }>[K];
+type Get<T, K extends PropertyKey> = Extract<T, { [K1 in K]: any }>[K];
 
 export interface MapStore<Value extends object = any> {
   /**

